src/GraphQL.php

```diff
@@ -418,7 +417,11 @@ public static function handleErrors(array $errors, callable $formatter): array
             }
 
             if (! $error instanceof Exception) {
-                $error = new FatalThrowableError($error);
+                $error = new \Exception(
+                    $error->getMessage(),
+                    $error->getCode(),
+                    $error
+                );
```

Comment:
I wrap the error with the Exception class to be backward-compatibile with `Laravel <= 6.x`.
I remove the `FatalThrowableError` because it doesn't exist anymore on Symfony 5